index.html

```diff
@@ -267,7 +267,64 @@ <h3>LET'S CONNECT</h3>
                     </div>
                 </div>
             </section>
+
         </main>
+        <footer>
+            <section class = "contact">
+                <div class="contact-1">
+                    <h1 id = "contact-me">
+                        Contact me
+                    </h1>
+                    <p id ="contact-messeage">
+                        If you have an application you <br>
+                         are interested in developing, a <br>
+                         feature that you need built or a <br>
+                          project that needs
+                        coding. I’d <br>
+                         love to help with it.
+                    </p>
+                    <form action="my-handling-form-page"
+                      method="post"
+                      class="my-contact-form">
```

Comment:
- [ ] Your form needs to be integrated with [formspree service](https://formspree.io/html), kindly check the link you'll get a starting guide on how to do that. 😁this also means that the `action` value should be similar to `https://formspree.io/f/{form_id}` 😉